modules/Core/src/main/java/org/terasology/core/world/generator/facetProviders/SurfaceObjectProvider.java

```diff
@@ -54,10 +55,11 @@ public void setSeed(long seed) {
 
     /**
      * Populates a given facet based on filters and population densities
-     * @param facet the facet to populate
+     *
+     * @param facet        the facet to populate
      * @param surfaceFacet the surface height facet
-     * @param typeFacet the facet that provides the environment
-     * @param filters a set of filters
+     * @param typeFacet    the facet that provides the environment
```

Comment:
IIRC I don't think we do and indentation has probably been inconsistently applied over time. I could go with either flavor, no preference